node/node.go

```diff
@@ -128,6 +137,41 @@ func New(conf *Config) (*Node, error) {
 	}, nil
 }
 
+func makeAccountManager(conf *Config) (*accounts.Manager, error) {
```

Comment:
i.e. make it 

`func (c *Config) AccountManager()  (*accounts.Manager, error)`